packages/@pollyjs/adapter-fetch/package.json

```diff
@@ -42,7 +42,9 @@
   "dependencies": {
     "@pollyjs/adapter": "^4.2.1",
     "@pollyjs/utils": "^4.1.0",
-    "detect-node": "^2.0.4"
+    "buffer": "^5.6.0",
+    "detect-node": "^2.0.4",
+    "to-arraybuffer": "^1.0.1"
```

Comment:
`to-arraybuffer` uses `require('buffer').Buffer` so I'm confused how this all is built in our current build pipeline - is it some how "browserifying" the node implementation and also bundling the npm "buffer" package?

Reply:
Ok so looked more into it to see what was actually being built. We current have [rollup-plugin-node-builtins](https://github.com/calvinmetcalf/rollup-plugin-node-builtins) setup which has support for using `Buffer` via [buffer-es6](https://www.npmjs.com/package/buffer-es6). 

If I imported the buffer via `import { Buffer } from 'buffer'` it only inline the polyfill for the UMD build but when I used `from 'buffer/'`, it inlined the polyfill for all 3 builds (CJS, ES, and UMD). Since this is only meant for the browser, it would make sense for it to be be polyfilled in all 3.

Reply:
Explains a lot, was very confused - thanks for looking into it